docs/running-on-kubernetes.md

```diff
@@ -1054,6 +1075,15 @@ See the [configuration page](configuration.html) for information on Spark config
   Specify the grace period in seconds when deleting a Spark application using spark-submit.
   </td>
 </tr>
+<tr>
+  <td>Client dependencies</td>
+  <td>Stores a local file to the path defined by <code>spark.kubernetes.file.upload.path</code></td>
+  <td>
+    Path to store files at the spark submit side in cluster mode. For example:
+    <code>spark.kubernetes.file.upload.path=s3a://<s3-bucket>/path</code>
+    File should specified as <code>client://path/to/file </code>
```

Comment:
Is it still necessary to use `client://` now? Or just enough to use `file://`?

Reply:
I changed the convention to file://. I will update the PR tomorrow with the integration tests based on ceph nano.